lib/beaker-hostgenerator/parser.rb

```diff
@@ -30,6 +30,9 @@ module Parser
     #   * host_settings    Any character (see `settings_string_to_map` for details)
     #                      Examples: {hostname=foo-bar, ip.address=123.4.5.6}
     #
+    #   * host_lists       Any character (see `settings_string_to_map` for details)
+    #                      Examples: [disks=16,8;other=foo,bar]
+    #
```

Comment:
This is no longer its own thing, right?  List support is now just a value within the host_settings component right?